desktops/kde.txt

```diff
@@ -0,0 +1,434 @@
+the breeze icon theme, and fonts. Alternatively, plasma is a meta-package that 
+includes things like an info center, a bug/crash reporting framework, additional
+plasma wallpapers, and bluedevil. It's basically KDE with all of the frills. Opt
+for plasma-desktop if you only want to selectively include these. For instance
+if you know you will not need bluetooth, you have no need for bluedevil. 
+
+
+    [1.1] Prerequisites
+    ____________________________________________________________________________
+
+    First, fetch the required repositories and add them to your $KISS_PATH. Omit
+    any repositories you already have, just ensure that community comes AFTER
+    KISS-kde/extra. These repositories may be stored anywhere; assume they are
+    in $HOME.
+
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ git clone https://github.com/kisslinux/community                     |
+    |   $ git clone https://github.com/dilyn-corner/KISS-kde                   |
+    |                                                                          |
+    |   $ export KISS_PATH="$HOME/KISS-kde/extra:$KISS_PATH"                   |
+    |   $ export KISS_PATH="$HOME/KISS-kde/plasma:$KISS_PATH"                  |
+    |   $ export KISS_PATH="$HOME/KISS-kde/frameworks:$KISS_PATH"              |
+    |   $ export KISS_PATH="$KISS_PATH:$HOME/community/community"              |
+    |                                                                          |
+    +--------------------------------------------------------------------------+
+
+    Next, enure that you have the proper required packages installed. These
+    include eudev, dbus, coreutils, and gnugrep. The former two are both runtime
+    dependencies, but the latter are both build-time requirements. As such, feel
+    free to remove them at the end if you wish! Additionally, libudev-zero has
+    not been tested here. Feel free to try it out.
+
+    If you want to get an idea for how long it will take to build the entire
+    plasma-desktop dependency tree, you can time the coreutils build and multiply
+    that time by ~800 to get a rough estimate of the time in minutes.
+    
+    +--------------------------------------------------------------------------+
+    |                                                                          |
+    |   $ kiss b eudev dbus coreutils gnugrep                                  |
+    |   $ kiss i eudev dbus coreutils gnugrep                                  |
+    |   $ kiss a coreutils /usr/bin/realpath                                   |
+    |   $ kiss a coreutils /usr/bin/mktemp                                     |
+    |   $ kiss a coreutils /usr/bin/ln                                         |
+    |   $ kiss a gnugrep   /usr/bin/grep                                       |
```

Comment:
Maybe explain why the `kiss a` commands are required to readers? 

(Also, can these gnu-isms be fixed? Am happy to help.)

Reply:
I can add that explanation.
If I am remembering right, `mktemp`, `grep`, and `ln` should be droppable. I just haven't worked on it (yet, was hoping to tackle it this week).

Reply:
Nice. Fine to keep it as-is for now (can always update it later). Feel free to add a little explanation for `kiss a` if you deem it necessary.

Reply:
I added a small reference!